projects/1.19/assets/client-commands/clientcommand/lang/zh_cn.json

```
@@ -0,0 +1,311 @@
+{
+  "commands.calias.notFound": "未找到别名“%s”",
+  "commands.calias.illegalFormatException": "指令格式错误或缺少参数",
+  "commands.calias.file.readError": "别名文件读取失败",
+  "commands.calias.file.writeError": "别名文件写入失败",
+  "commands.calias.addAlias.aliasAlreadyExists": "别名“%s”已经存在",
+  "commands.calias.addAlias.commandAlreadyExists": "指令“/%s”已经存在",
+  "commands.calias.addAlias.success": "成功加入别名“%s”",
+  "commands.calias.listAliases.noAliasesRegistered": "没有已注册的别名",
+  "commands.calias.listAliases.success": "%s已注册的别名：",
+  "commands.calias.removeAlias.success": "成功移除别名“%s”",
+
+  "commands.careastats.notLoaded": "该区域的区块未加载",
+  "commands.careastats.output.chunksScanned": "扫描%d个区块，用时%dns≈%dms",
+  "commands.careastats.output.blocksMatched": "在一共%2$d个方块中匹配到%1$d个方块",
+  "commands.careastats.output.entitiesFound": "在此区域中找到%d个实体",
+
+  "commands.cbook.success": "成功编辑书本",
+  "commands.cbook.commandException": "你没有手持书本",
+
+  "commands.ccalc.expected": "期望的标识符%s",
+  "commands.ccalc.invalidArgumentCount": "函数“%s”无法接受%d个自变量",
+  "commands.ccalc.parse": "表达式解析结果：%s",
+  "commands.ccalc.parse.binaryOperator.addition": "(%s + %s)",
+  "commands.ccalc.parse.binaryOperator.division": "(%s / %s)",
+  "commands.ccalc.parse.binaryOperator.exponentiation": "(%s ^ %s)",
+  "commands.ccalc.parse.binaryOperator.modulo": "(%s % %s)",
+  "commands.ccalc.parse.binaryOperator.multiplication": "(%s * %s)",
+  "commands.ccalc.parse.binaryOperator.subtraction": "(%s - %s)",
+  "commands.ccalc.parse.constant": "(常量‘%s’)",
+  "commands.ccalc.parse.function": "(函数‘%s’带有参数：%s)",
+  "commands.ccalc.parse.literal": "(字面量：%s)",
+  "commands.ccalc.parse.negate": "(负值：%s)",
+  "commands.ccalc.tooDeeplyNested": "表达式嵌套过深",
+
+  "commands.ccalcstack.success": "%d个%s是%d组余%d个",
+  "commands.ccalcstack.success.empty": "%d个物品是%d组余%d个",
+  "commands.ccalcstack.success.empty.exact": "%d个物品是正好%d组",
+  "commands.ccalcstack.success.exact": "%d个%s是正好%d组",
+
+  "commands.ccrackrng.failed": "破解玩家随机种子失败",
+  "commands.ccrackrng.failed.help": "提示：RNG控制并不在一些被修改过的服务器有效，特别是Paper端。",
+  "commands.ccrackrng.retries": "正在破解玩家随机种子，尝试次数：%d/%d",
+  "commands.ccrackrng.starting": "正在破解玩家随机种子",
+  "commands.ccrackrng.success": "成功破解玩家随机种子：%d",
+
+  "commands.cenchant.expectedWithWithoutExactly": "有效参数：“with”“without”“exactly”",
+  "commands.cenchant.failed": "你请求的附魔无法获得或者需要太长时间",
+  "commands.cenchant.incompatible": "互斥的魔咒",
+  "commands.cenchant.needEnchantingPrediction": "本命令需要启用附魔预测",
+  "commands.cenchant.success": "正在开始操控魔咒",
+  "commands.cenchant.uncracked": "你需要先破解随机种子",
+
+  "commands.cfind.found.left": "在",
+  "commands.cfind.found.right": " 找到%1$s, 距离你%2$s格",
```

```suggestion
  "commands.cfind.found.right": "找到%1$s, 距离你%2$s格",
```
不知道这里实际的表现是怎么样的
可以放张截图之类的（）

---

image.png

---

找到一个匹配查找条件的实体
在（x,y,z）找到【实体名称】，距离你【距离】格

---

记得删空格

---

记得删空格

---

记得删空格啊